zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/common/CatchEventBehavior.java

```diff
@@ -161,19 +161,26 @@ public Either<Failure, Void> subscribeToEvents(
    * @param context the context of the element instance that subscribes to events
    * @param supplier the supplier of catch events to subscribe to, typically the element of the
    *     element instance that subscribes to events
-   * @param filter the filter for catch events to subscribe to, only events that match the filter
-   *     are subscribed to.
+   * @param filterBeforeEvaluation the filter apply for catch events before applying any
+   *     evaluations. This is especially useful for filtering catch events that doesn't require an
+   *     expression evaluation.
+   * @param filterAfterEvaluation the filter for catch events to subscribe to, only events that
+   *     match the filter are subscribed to.
```

Comment:
🔧 Let's align the descriptions of the parameters

```suggestion
   * @param filterBeforeEvaluation the filter for catch events to subscribe to. Only events that
   *     match the filter are subscribed to. This filter is applied before evaluating the catch
   *     event's expressions. This is especially useful for filtering catch events that doesn't
   *     require an expression evaluation.
   * @param filterAfterEvaluation the filter for catch events to subscribe to. Only events that
   *     match the filter are subscribed to. This filter is applied after evaluating the catch
   *     event's expressions.
```